tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -333,9 +333,14 @@ TfLiteStatus TestFullyConnectedQuantized(
 
 TF_LITE_MICRO_TESTS_BEGIN
 
-#if !defined(XTENSA)  // TODO(b/170503075): xtensa kernels are less general than
-                      // reference kernels and we ifdef out test cases that are
-                      // currently known to fail.
+#if !defined(XTENSA) && !defined(CEVA_BX1) && \
+    !defined(CEVA_SP500)  // TODO(b/170503075): xtensa kernels are less general
+                          // than
+// reference kernels and we ifdef out test cases that are
+// currently known to fail.
+// CEVA's fully connected implementation assumes weights_zero_point=0 as
+// described in TFLite's quantization specification. tests which use a different
+// zero point will so ifdefed out.
```

Comment:
Can you move these comments to their own paragraphs, one each for Xtensa and CEVA and link to the github issue as well for the asymmetric quantization?

```cc 
#if !defined(XTENSA) && !defined(CEVA_BX1) && !defined(CEVA_SP500)
// TODO(b/170503075): xtensa kernels are less general than
// reference kernels and we ifdef out test cases that are
// currently known to fail.

// TODO(#44912): Remove uint8 support from the existing fully connected kernels and also remove the uint8 test cases. CEVA's fully connected implementation assumes weights_zero_point=0 as
// described in TFLite's quantization specification. Tests which use the deprecated asymmetric quantization for weights are ifdefed out.
```